librz/core/cgraph.c

```diff
@@ -536,7 +537,6 @@ RZ_API RzCoreGraphFormat rz_core_graph_format_from_string(RZ_NULLABLE const char
  * \brief Convert a string to RzCoreGraphType.
  */
 RZ_API RzCoreGraphType rz_core_graph_type_from_string(RZ_NULLABLE const char *x) {
-	rz_return_val_if_fail(x, RZ_CORE_GRAPH_TYPE_UNK);
 	if (strcmp(x, "dataref") == 0) {
```

Comment:
I think `strcmp(NULL)` would crash. You probably want to check for NULL before calling it.